src/main/java/com/locallampoon/fiveh/core/Game.java

```diff
@@ -140,10 +140,22 @@ static void implementCommandTwoWords(List<String> parsedCommandList, List<String
                     default -> actionPanel.appendTextArea("Invalid Action",FG_COLOR);
                 }
             }
+            case "enter" -> {
+                if (parsedCommandList.get(1).equalsIgnoreCase("passage") && Game.checkWinCondition()) {
+                    playerCurrentRoom.setDesc("""
+                    \tA passageway is opened to you and it glows a warm blue, like the keys in your pack, brighter and brighter as you approach. Its warmth lifts the spirits and fills your wearied mind with hope... but each time you pass through it - nothing. You transport to the same place. It matters not how many times you cross.  
+                    \tThe realization that dawns upon you is as clear as it is maddening and final. There is no end. There is no beginning. There is no escape. This is a prison. 
```

Comment:
Upon reading your final ending of our game, I think I just threw up in my mouth.